grove/vendor/github.com/coreos/bbolt/Makefile

```diff
@@ -1,30 +0,0 @@
-BRANCH=`git rev-parse --abbrev-ref HEAD`
```

Comment:
Did this vendored file need to be deleted?

Reply:
That vendored change was a result of accidentally committing the changes from #4867 to this branch. I thought those should all be dropped after a rebase, though.... Lemme fix

Reply:
Fixed